pycloudflare/exceptions.py

```diff
@@ -0,0 +1,2 @@
+class SSLUnavailable(Exception):
```

Comment:
I think we should describe how this can happen. In a doc string, possibly?

If you haven't verified a CNAME zone within the grace period (a week), it can't be verified any more.